customer_outstanding_statement/report/customer_outstanding_statement.py

```diff
@@ -94,8 +94,9 @@ def _display_lines_sql_q1(self, partners, date_end, account_type):
 
     def _display_lines_sql_q2(self):
         return """
-            SELECT partner_id, currency_id, move_id, date, date_maturity,
-                            debit, credit, name, ref, blocked, company_id,
+            SELECT Q1.partner_id, Q1.currency_id, Q1.move_id, 
+            Q1.date, Q1.date_maturity, Q1.debit, Q1.credit, 
+            Q1.name, Q1.ref, Q1.blocked, Q1.company_id,
```

Comment:
I think this change is not needed, there is no joint, thus no problems. But if you want to proceed with this change, then you should also put the Q1 in the `currency_id` of the line below and fix the related flake8.

Reply:
Fixed.

Reply:
Fixed

Reply:
still flake8 fails 🙁

Reply:
why is it Q1 in a function called q2? am I just being stupid?

Reply:
@mreficent IRT flake8, I have a more general problem with formatting SQL syntax to fit python style. Using triple quoted strings and laying it out nicely makes for easy reading of python code, but it generates truly horrible to read SQL logs. Honestly, I feel like we are better with flake8 overrides, but just a personal opinion.

But if you are going to lay it out for python, then you should indent continuation lines of major clauses (e.g. Q1.date and Q1.name should be indented) for readability.

Reply:
Previous flake8 error was that there was blank spaces and the end of some lines